k9mail/src/main/java/com/fsck/k9/notification/NotificationHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.fsck.k9.notification;
+
+
+class NotificationHolder {
```

Comment:
Agreed.

Any suggestions for better names? Rename `NotificationsHolder` to `NotificationData`?